doc/best_practices.md

```diff
@@ -0,0 +1,19 @@
+# Best practices
+
+## Large-scale deployment
+
+To run etcd cluster at large-scale, it is important to assign etcd pods to nodes with desired resources, such as SSD, high performance network. 
+
+### Assign to nodes with desired resources
+
+Kuberentes nodes can be attached with labels. Users can [assign pods to nodes with given labels](http://kubernetes.io/docs/user-guide/node-selection/). Similarly for the etcd-operator, users can specify `Node Selector` in the cluster spec to select nodes that etcd pods will be assigned to. For example, users can label a set of nodes with SSD with label `"disk"="SSD"`. To assign etcd pods to these node, users can specify `"disk"="SSD"` in the cluster spec.
```

Comment:
"to select nodes that etcd pods will be assigned to" ->
"to select nodes for etcd pods".